evoting/lib/election.go

```diff
@@ -118,23 +118,25 @@ func (e *Election) setVoted(s *skipchain.Service, user uint32) error {
 }
 
 func (e *Election) setStage(s *skipchain.Service) error {
-	db := s.GetDB()
-	latest, err := db.GetLatest(db.GetByID(e.ID))
+	target := 2 * len(e.Roster.List) / 3
+	partials, err := e.Partials(s)
 	if err != nil {
-		return errors.New("error getting latest skipblock")
+		return err
 	}
-	transaction := UnmarshalTransaction(latest.Data)
-	if transaction == nil {
-		return errors.New("invalid transaction")
+	if len(partials) > target {
```

Comment:
Why is that not `>=`?

Reply:
because the minimum required count is floor(2n/3) + 1? Considering 7 conodes, `2*7/3 == 4` and we need at least 5

Reply:
OK - so you need either a comment here to explain what you wrote, or initialize `target` with `2*n/3+1`. Also, we usually call it `threshold`.
Sorry for nitpicking, but I'd like this code to be understandable also later on!

Reply:
Fixed in the new commit 👍